app/scaffold-controllers/navigationController.js

```diff
@@ -3,7 +3,9 @@ define([
     'application',
     'plugins/anchoredLayoutPlugin',
     'plugins/navigationPlugin',
-    'scaffold-controllers/navigationHeaderController'
+    'scaffold-controllers/navigationHeaderController',
+    'scaffold-controllers/searchBarController',
+    'scaffold-config/searchConfig'
```

Comment:
Nitpick: I believe `scaffold-config` was mapped to `config`, perhaps we should use that for consistency :O